src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -275,6 +276,7 @@ private void startJnlpProc() throws Exception {
         });
     }
 
+    @Issue("JENKINS-41854")
```

Comment:
Rather write a _new_ test case, and mark it

```java
@Ignore("TODO currently fails with: …")
@Issue("JENKINS-41854")
```

Reply:
I've copied and pasted the existing test case into a new test case and marked it as suggested, leaving the existing test case untouched.